### ISSUE TITLE
Health Monitors unnecessary updates in BIG-IP v13.0

### ISSUE THREAD

**Issue by sjberman** [F5Networks/k8s-bigip-ctlr#502]
Health monitors are being updated too often on BIG-IP v13.0

```
0:43:09 2018/01/23 17:41:59 [INFO] Starting: Version: 1.4.0-WIP, BuildInfo: n628-332372230
10:43:09 2018/01/23 17:41:59 [INFO] ConfigWriter started: 0xc4202150e0
10:43:09 2018/01/23 17:41:59 [INFO] Started config driver sub-process at pid: 15
10:43:09 2018/01/23 17:41:59 [INFO] NodePoller (0xc42023a090) registering new listener: 0x4070c0
10:43:09 2018/01/23 17:42:00 [INFO] NodePoller started: (0xc42023a090)
10:43:09 2018/01/23 17:42:00 [INFO] Wrote 0 Virtual Server and 0 IApp configs
10:43:09 2018/01/23 17:42:01 [INFO] Wrote 1 Virtual Server and 0 IApp configs
10:43:09 2018/01/23 17:42:13 [INFO] [2018-01-23 17:42:13,008 __main__ INFO] entering inotify loop to watch /tmp/k8s-bigip-ctlr.config235152314/config.json
10:43:09 2018/01/23 17:42:14 [INFO] [2018-01-23 17:42:14,616 f5_cccl.resource.resource INFO] Creating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:14 [INFO] [2018-01-23 17:42:14,709 f5_cccl.resource.resource INFO] Creating ApiPool: /test/cfgmap_default_svc-1-map_svc-1
10:43:09 2018/01/23 17:42:14 [INFO] [2018-01-23 17:42:14,809 f5_cccl.resource.resource INFO] Creating ApiInternalDataGroup: /test/https_redirect_dg
10:43:09 2018/01/23 17:42:15 [INFO] [2018-01-23 17:42:14,930 f5_cccl.resource.resource INFO] Creating ApiIRule: /test/http_redirect_irule
10:43:09 2018/01/23 17:42:15 [INFO] [2018-01-23 17:42:15,009 f5_cccl.resource.resource INFO] Creating ApiVirtualServer: /test/default_svc-1-map
10:43:09 2018/01/23 17:42:16 [INFO] [2018-01-23 17:42:16,017 f5_cccl.resource.resource INFO] Creating ApiArp: /test/k8s-10.244.1.37
10:43:09 2018/01/23 17:42:16 [INFO] [2018-01-23 17:42:16,132 f5_cccl.resource.resource INFO] Updating ApiFDBTunnel: /test/vxlan-tunnel-mp
10:43:09 2018/01/23 17:42:19 [INFO] Wrote 2 Virtual Server and 0 IApp configs
10:43:09 2018/01/23 17:42:19 [WARNING] Overwriting existing entry for backend {ServiceName:svc ServicePort:80 Namespace:default}
10:43:09 2018/01/23 17:42:19 [INFO] Wrote 2 Virtual Server and 0 IApp configs
10:43:09 2018/01/23 17:42:19 [INFO] [2018-01-23 17:42:19,524 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:21 [INFO] [2018-01-23 17:42:21,808 f5_cccl.resource.resource INFO] Creating ApiPool: /test/ingress_default_svc
10:43:09 2018/01/23 17:42:21 [INFO] [2018-01-23 17:42:21,908 f5_cccl.resource.resource INFO] Creating ApiVirtualServer: /test/ingress_10-10-10-10_80
10:43:09 2018/01/23 17:42:21 [INFO] [2018-01-23 17:42:21,942 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:23 [INFO] [2018-01-23 17:42:23,209 f5_cccl.resource.resource INFO] Creating ApiArp: /test/k8s-10.244.1.38
10:43:09 2018/01/23 17:42:24 [INFO] [2018-01-23 17:42:24,110 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:26 [INFO] [2018-01-23 17:42:26,208 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:28 [INFO] [2018-01-23 17:42:28,808 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:31 [INFO] [2018-01-23 17:42:31,408 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:33 [INFO] [2018-01-23 17:42:33,410 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:35 [INFO] [2018-01-23 17:42:35,510 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:37 [INFO] [2018-01-23 17:42:37,610 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:39 [INFO] [2018-01-23 17:42:39,719 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
10:43:09 2018/01/23 17:42:41 [INFO] [2018-01-23 17:42:41,818 f5_cccl.resource.resource INFO] Updating ApiHTTPMonitor: /test/cfgmap_default_svc-1-map_svc-1_0_http
```

#### Kubernetes Version
1.7

#### Controller Version
1.4

#### BIG-IP Version
13.0

**Comment by edarzins** [comment]
This test fails for BIG-IP v13.0 because that version enforces a relationship between the health monitor interval and timeout values (interval must be less than timeout).

**Comment by edarzins** [comment]
https://github.com/f5devcentral/f5-cccl/issues/208

**Comment by edarzins** [comment]
Fixed via 9301c67a66a58ebc67406668f328fe2ad289b7c4.
